src/api/app/models/staging/workflow.rb

```diff
@@ -17,7 +17,8 @@ def without_staged_requests
 
   has_many :target_of_bs_requests, through: :project, foreign_key: 'staging_workflow_id' do
     def stageable
-      in_states(['new', 'review'])
+      managers_group_title = proxy_association.owner.managers_group.title
```

Comment:
As the group can be deleted and then nullified the association... why don't you use `try(:title)` instead? or assign an empty string in that case